src/components/Pages/LoginPage/utils.ts

```diff
@@ -97,3 +98,8 @@ export async function getIdentitySignature(address: string, provider: Provider):
 
   return identity
 }
+
+export function getIsMobile() {
```

Comment:
Use media queries from decentraland-ui instead https://github.com/decentraland/ui/blob/af2045c96c7b22a898ebd0a03b726f054a30ae1a/src/components/Media/Media.ts#L28

Reply:
This is a device type dependent config, independent of the canvas size